libbeat/outputs/otelconsumer/otelconsumer_test.go

```diff
@@ -113,7 +114,7 @@ func TestPublish(t *testing.T) {
 		var subFields = []string{"dataset", "namespace", "type"}
 		for _, subField := range subFields {
 			gotValue, ok := attributes.Get("data_stream." + subField)
-			require.True(t, ok)
+			require.True(t, ok, fmt.Sprintf("data_stream.%s not found on log record attribute", subField))
```

Comment:
[NIT]
You can use `Truef` to format
```suggestion
			require.Truef(t, ok, "data_stream.%s not found on log record attribute", subField)
```